Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_render_utils.mm

```diff
@@ -787,8 +787,9 @@ void SetupCommonBlitWithDrawStates(const gl::Context *context,
         GetViewport(params.dstRect, params.dstTextureSize.height, params.dstFlipY);
     MTLScissorRect scissorRectMtl =
         GetScissorRect(params.dstScissorRect, params.dstTextureSize.height, params.dstFlipY);
+    // TODO:
```

Comment:
How big is this TODO to handle? Please reformat to `TODO(djg): ...` and describe more why the rasterization rate map isn't available here.